mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -982,19 +989,33 @@ class LogisticRegressionModel private[spark] (
     throw new SparkException("No training summary available for this LogisticRegressionModel")
   }
 
+  @Since("2.2.0")
+  def binarySummary: BinaryLogisticRegressionTrainingSummary = summary match {
```

Comment:
Could I ask why we need a separate ```binarySummary```? If we get a binary classification model, the ```trainingSummary ``` should be the corresponding binary summary. Is that correct?